lib/src/widgets/buttons/ds_button.widget.dart

```diff
@@ -0,0 +1,121 @@
+import 'package:blip_ds/blip_ds.dart';
+import 'package:flutter/material.dart';
+
+/// A container that has some default properties which should be extended by others Design System's [ButtonStyleButton].
+class DSButton extends StatelessWidget {
+  final void Function()? onPressed;
+  final Color backgroundColor;
+  final Color foregroundColor;
+  final Color? borderColor;
+  final Icon? leadingIcon;
+  final String? label;
+  final Icon? trailingIcon;
+  final bool disable;
+  final bool loading;
+
+  final List<Widget> _contentList = [];
+
+  /// Creates a Design System's [ButtonStyleButton].
+  ///
+  /// Set [disable] to deactivate this button's [onPressed] and change his style.
+  /// Set [loading] to override content with a loading animation. It also deactivates [onPressed].
+  DSButton({
+    required this.onPressed,
+    required this.backgroundColor,
+    required this.foregroundColor,
+    super.key,
+    this.leadingIcon,
+    this.label,
+    this.trailingIcon,
+    this.borderColor,
+    this.disable = false,
+    this.loading = false,
+  });
+
+  @override
+  Widget build(BuildContext context) {
+    return OutlinedButton(
+      onPressed: !disable || !loading ? onPressed : null,
```

Comment:
I'm using the disable property, but the onPressed function keeps getting called. Maybe you should change the OR operator to an AND operator.